config.json.example

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I like the idea of having an empty config to supplement the example. I am torn as to whether it should be kept in the /Examples directory tho

Reply:
I think its fine for it to be the project root because that is where the actual `config.json` gets read from, however I also think the examples should be in the readme instead.

Reply:
Ideally the features of the readme indicate what each element of the config is responsible for so I think that's fine